### PR TITLE
Add pagination to results table in index.php

### PR DESCRIPTION
**Pull request by jraddaoui**
Add default pagination values and modify select query to use
them. Build and add a Bootstrap pagination component, currently
with all available pages.

Also, fix table ending and remove totals from index.php

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -159,7 +176,19 @@
...
+
+	// End if there is no results
+	if (!$total)
+	{
+		echo 'No results';
+		exit;
+	}
+
+	// Get results with pagination
+	$query = $db->query('SELECT * FROM unit LIMIT ' . $skip . ', ' . $limit);
```
Comment (mcantelon) [inline index.php]: Minor nitpck: could use sprintf.

File: index.php [hunk 2]
```diff
@@ -107,6 +107,23 @@
 		$selectedDB = '/var/archivematica/automation-tools/transfers-3.db';
 	};
 
+	// Pagination defaults
+	$limit = 20;
+	$page = 1;
+
+	// Update pagination values from request
+	if (isset($_GET['limit']) && is_numeric($_GET['limit']))
```
Comment (mcantelon) [inline index.php]: Minor nitpick: Could use ctype_digit.

**Comment by mcantelon** [comment]
Looks good to me (only one minor nitpick).

**Comment by jraddaoui** [comment]
Thanks Mike! It should be better now

**Pull request merged**